pkg/testworkflows/testworkflowprocessor/action/process.go

```diff
@@ -0,0 +1,152 @@
+package action
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/kubeshop/testkube/internal/common"
+	"github.com/kubeshop/testkube/pkg/expressions"
+	"github.com/kubeshop/testkube/pkg/testworkflows/testworkflowprocessor/action/actiontypes"
+	"github.com/kubeshop/testkube/pkg/testworkflows/testworkflowprocessor/action/actiontypes/lite"
+	stage2 "github.com/kubeshop/testkube/pkg/testworkflows/testworkflowprocessor/stage"
+)
+
+func process(currentStatus string, parents []string, stage stage2.Stage, machines ...expressions.Machine) (actions []actiontypes.Action, err error) {
+	// Store the init status
+	actions = append(actions, actiontypes.Action{
+		CurrentStatus: common.Ptr(currentStatus),
+	})
+
+	// Compute the skip condition
+	condition := stage.Condition()
+	if condition == "" || condition == "null" {
+		condition = "passed"
```

Comment:
should be const

Reply:
It rather shouldn't - `passed` is just a built-in variable, and it could be any kind of expression